README.md

```diff
@@ -536,7 +536,19 @@ Return the time and date that the response was sent.
 
 ### `Fieldhand::NetworkError`
 
-An error (descended from `StandardError`) to represent any network issues encountered during interaction with the repository. Any underlying exception is exposed in Ruby 2.1 onwards through [`Exception#cause`](https://ruby-doc.org/core-2.1.0/Exception.html#method-i-cause).
+An error (descended from `StandardError`) to represent any network issues encountered during interaction with the repository.  
+If the HTTP request is not successful (returning a status code other than 200), 
+a `NetworkError` exception will be raised containing the error message and the response object.  
+Any underlying exception is exposed in Ruby 2.1 onwards through [`Exception#cause`](https://ruby-doc.org/core-2.1.0/Exception.html#method-i-cause).
+
+#### Fieldhand::NetworkError.new(message[, response])
```

Comment:
Should this be part of the public API (that is, do we expect clients to initialise their own errors) or should we only document the public `response` getter?